deps/rabbit/src/rabbit_classic_queue.erl

```diff
@@ -502,15 +521,15 @@ recover_durable_queues(QueuesAndRecoveryTerms) ->
 capabilities() ->
     #{unsupported_policies => [%% Stream policies
                                <<"max-age">>, <<"stream-max-segment-size-bytes">>,
-                               <<"queue-leader-locator">>, <<"initial-cluster-size">>,
```

Comment:
Can we add an integration that the new `queue-leader-locator` policy actually works with classic queues?